src/app/calculator/calculator-form/calculator-form-individual/calculator-form-individual.component.ts

```diff
@@ -15,6 +15,7 @@ export class CalculatorFormIndividualComponent implements OnInit, OnDestroy {
   resSub: Subscription;
   errAddress: boolean = false;
   errSub: Subscription;
+  genders = ['male', 'female'];
```

Comment:
V typescriptu kdyžtak existují enumy, eliminuje to případné překlepy.
Můžeš si vytvořit soubor gender.enum.ts
`export enum GenderEnum {
Male = 'male',
Female = 'female'
}`

Co se týče potom použití, tak akorát nahradíš 'male' (resp. 'female') tím enumem GenderEnum.Male (resp. GenderEnum.Female).
V html to použít nejde takhle jednoduše, ale dá se to případně taky.